File: api/src/app/Poi.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

/**
 * @property integer id
 * @property string|null id_sitra1
 * @property string|null type
 * @property string|null type_detail
 * @property string|null name
 * @property string|null address
 * @property integer|null zip_code
 * @property string|null town
 * @property string|null phone
 * @property string|null fax
 * @property string|null fax_phone
 * @property string|null email
 * @property string|null website
 * @property string|null facebook
 * @property string|null rank
 * @property string|null opening_times
 * @property string|null price
 * @property float|null pricemin
 * @property float|null pricemax
 * @property string|null author
 * @property integer|null gid
 * @property \DateTime creation_date
 * @property \DateTime last_update
 * @property \DateTime last_update_fme
 * @property double latitude
 * @property double longitude
 * @method static count()
 * @method static findOrFail($id)
 */
class Poi extends Model
{
    protected $table = 'poi';
    public $incrementing = false;
    public $timestamps = false;
}

File: app/app/src/main/java/com/itakademy/grandlyon/model/Poi.kt
package com.itakademy.grandlyon.model

import java.util.*

class Poi(
    val id : Int,
    val id_sitra1 : String,
    val type: PoiType,
    val type_detail: String,
    val name: String,
    val address: String,
    val zip_code: Int?,
    val town: String,
    val phone: String,
    val fax: String,
    val fax_phone: String,
    val email: String,
    val website: String,
    val facebook: String,
    val rank: Int?,
    val opening_times: String,
    val price: String,
    val pricemin: Float?,
    val pricemax: Float?,
    val author: String,
    val gid: Int?,
    val creation_date: Date?,
    val last_update: Date?,
    val last_update_fme: Date?,
    val geometry: Geometry?
)

@Suppress("SpellCheckingInspection")
enum class PoiType {
    HEBERGEMENT_LOCATIF,
    PATRIMOINE_CULTUREL,
    RESTAURATION,
    COMMERCE_ET_SERVICE,
    EQUIPEMENT,
    HOTELLERIE,
    DEGUSTATION,
    HOTELLERIE_PLEIN_AIR,
    HEBERGEMENT_COLLECTIF,
    PATRIMOINE_NATURE
}

class Geometry(
    val longitude: Double?,
    val latitude: Double?
)